backend/Tasque.BLL/Services/ProjectService.cs

```diff
@@ -389,4 +390,24 @@ public List<TaskStateDto> GetProjectStatesById(int projectId)
     {
         return _mapper.Map<List<TaskStateDto>>(_db.TaskStates.Where(s => s.ProjectId == projectId));
     }
+
+    public async Task<List<ProjectCardDTO>> GetProjectCardsByUserId(int userId)
+    {
+        var user = await _db.Users
```

Comment:
We take Id by email from the DB, confirming user's existence. Under no circumstances user cannot be found.

Reply:
It prevents losing connection to db.